libsolidity/codegen/ir/IRGenerator.h

```diff
@@ -56,6 +56,11 @@ class IRGenerator
 	std::string generate(ContractDefinition const& _contract);
 	std::string generate(Block const& _block);
 
+	/// Collects generated code for contract functions. May skip functions that never get called
```

Comment:
May skip or skips? What is the exact behavior?

Reply:
It wasn't skipping in the commit where I added it and then I changed the behavior so that it skips them :)

I'm not sure I should promise that it always skips them since not doing it does not hurt anything and the behavior could later change. Such comments easily go out of sync with the code.